pom.xml

```diff
@@ -100,6 +100,13 @@
             <version>3.2</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+        	<groupId>org.apache.sling</groupId>
+        	<artifactId>org.apache.sling.testing.sling-mock</artifactId>
+        	<version>2.2.12</version>
+        	<type>bundle</type>
```

Comment:
please remove this type specifier - its not required.

Reply:
sure thing